external/aliases/aliases.go

```diff
@@ -78,6 +78,54 @@ func SaveAliasesForResources(jsonTxt string) {
 
 }
 
+func DeleteAliasesById(id string, jsonApiType string) {
+	profileDirectory := GetAliasDataDirectory()
+	filelock.Lock()
+	defer filelock.Unlock()
+
+	aliasFile := getAliasFileForJsonApiType(profileDirectory, jsonApiType)
+	data, err := ioutil.ReadFile(aliasFile)
+	if err != nil {
+		log.Debugf("Could not read %s, error %s", aliasFile, err)
+		data = []byte{}
+	} else {
+	}
+
+	aliasMap := map[string]string{}
```

Comment:
I don't think we need all this duplication is my guess. Can we refactor this file a bit so that we don't need to do all of this.